src/app/services/data/trainrunsection.service.ts

```diff
@@ -341,6 +342,7 @@ export class TrainrunSectionService implements OnDestroy {
     targetArrival: number,
     targetDeparture: number,
     travelTime: number,
+    emit: boolean = true
```

Comment:
Not here : https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/blob/add-new-operations-event-emitters/src/app/services/data/trainrun.service.ts#L249

Because the parent function is called by another service, it would lead to multiple emission (actually maybe that this whole function could be refactor, because not only the emission is called many times, but not the scope of this feature)

Reply:
Ah, indeed, missed it in the diff!